### ISSUE TITLE
Indicate cluster name in shell prompt

### ISSUE THREAD

**Issue by kelvk** [bloomberg/chef-bcpc#905]
I think @erhudy first suggested this. We should do this to avoid confusion as the number of clusters grow.

**Comment by mihalis68** [comment]
Agreed

**Comment by mihalis68** [comment]
We could make  a file /etc/clustername contain the name and then set the prompt like this :
```
export PS1='`cat /etc/clustername ` [\u@\h \W]\$ '
```
and so we'll get prompts like
```
DOB1 [ubuntu@ob-cloud-r01an12 ~]$ 
```

**Comment by erhudy** [comment]
Yeah, I was initially thinking about doing it in Ansible but it would make more sense to just do it in Chef. Whatever we do, we should ensure that it's also visible even if you `sudo su -` to root.

**Comment by kamidzi** [comment]
Yeah, we should just configure it system-wide instead of for root and operations

**Comment by kamidzi** [comment]
This, btw, isn't as simple as just configuring /etc/profile. Shell-specific rc files may override the prompt value; For example /etc/bashrc. http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/trusty/bash/trusty/view/head:/debian/etc.bash.bashrc?start_revid=73#L19

**Comment by erhudy** [comment]
Indeed, that became apparently pretty quickly, which is probably why nobody's PR'd this yet. :anger:

**Comment by kamidzi** [comment]
I'm actually thinking this might not belong in chef after all - I mean if we have to configure environment specific users like operations, maybe ansible is the right place?

**Comment by erhudy** [comment]
Editing the operations user environment template is probably a good first step.
